packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -787,8 +787,8 @@ describe('build', () => {
     expect(files.sort()).toEqual(['index.html', 'package.json']);
   });
 
-  it('should set `VERCEL_ANALYTICS_ID` environment variable', async () => {
-    const cwd = fixture('vercel-analytics');
+  it('should set `VERCEL_ANALYTICS_ID` environment variable if Vercel Speed Insights is enabled', async () => {
```

Comment:
**suggestion:** could you change to `should set VERCEL_ANALYTICS_ID environment variable if @vercel/speed-insights is not used`?

Reply:
It should always set it if it's enabled. The other part is tested in each builder where we unset it if the package is installed